hoomd/mpcd/ParticleData.h

```diff
@@ -429,9 +428,6 @@ class PYBIND11_EXPORT ParticleData
         GPUArray<Scalar4> m_pos_alt;        //!< Alternate position array
         GPUArray<Scalar4> m_vel_alt;        //!< Alternate velocity array
         GPUArray<unsigned int> m_tag_alt;   //!< Alternate tag array
-        #ifdef ENABLE_MPI
```

Comment:
Can you please revert this change here, and just move the `#endif // ENABLE_MPI` after line 433? That will be a smaller change and keep the variables organized together.

You would no longer need the changes below.